<ui version="4.0" >
 <class>PreviewConfigurationWidget</class>
 <widget class="QGroupBox" name="PreviewConfigurationWidget" >
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="title" >
   <string>Print/Preview Configuration</string>
  </property>
  <property name="checkable" >
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="m_styleLabel" >
     <property name="text" >
      <string>Style</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="m_styleCombo" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="m_appStyleSheetLabel" >
     <property name="text" >
      <string>Style sheet</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="qdesigner_internal::TextPropertyEditor" name="m_appStyleSheetLineEdit" >
       <property name="minimumSize" >
        <size>
         <width>149</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_appStyleSheetChangeButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_appStyleSheetClearButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="m_skinLabel" >
     <property name="text" >
      <string>Device skin</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QComboBox" name="m_skinCombo" />
     </item>
     <item>
      <widget class="QToolButton" name="m_skinRemoveButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qdesigner_internal::TextPropertyEditor</class>
   <extends>QLineEdit</extends>
   <header location="global" >textpropertyeditor_p.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
